rotkehlchen/chain/ethereum/modules/balancer/typing.py

```diff
@@ -251,14 +251,29 @@ def deserialize_from_db(
             amounts=amounts,
         )
 
-    def serialize(self) -> Dict[str, Any]:
+    def serialize(
+            self,
+            pool_tokens: Optional[List[BalancerBPTEventPoolToken]] = None,
+    ) -> Dict[str, Any]:
+        amounts: Union[List[str], Dict[str, Any]]
+        if isinstance(pool_tokens, list) and len(pool_tokens) > 0:
+            # Excludes assets with zero amount
```

Comment:
As discussed in discord please don't go into extra lenghts to exclude zero amounts. There is no reason.

Also amend the docs for this.